_en/0001-01-01-membership.html

```diff
@@ -10,36 +10,39 @@
 <div class="row">
     <div class="col-md-8 col-md-offset-2 center-justified">
         <p>
-            javaBin is open for everyone who wants to be a member. Being a member with a paid membership means that you support
-            the Java community in Norway and helping us organize meetups and JavaZone.
+            javaBin is open for everyone who wants to become a member. Being a paying member means that you support
+            the Java community in Norway and help us organize various events throughout the year, and JavaZone.
         </p>
 
         <h3 class="text-muted">
-            How much costs a membership?
+            How much does a membership cost?
         </h3>
 
         <p>
             Membership in javaBin is included in the JavaZone ticket, so if you have bought a ticket to JavaZone or have received this
-            by your employer, then you are already a member of javaBin for one year. If you are not going to JavaZone, you can buy a single membership.
-            Price for this is 500 NOK for 12 months membership. Students with a valid student ID can apply for free membership.
-
-            If you want student membership, please send an email to <a href="mailto:medlemskap@java.no">medlemskap@java.no</a>.
+            from your employer, then you are already a member of javaBin for that year. If you are not going to JavaZone, you can 
+            buy a separate membership. Price for this is 500 NOK for a 12-month membership. 
```

Comment:
If you are not going to JavaZone then you can buy a separate 12-month membership for 500 NOK.